examples/substreams-powered-subgraph/substreams.yaml

```diff
@@ -0,0 +1,33 @@
+specVersion: v0.1.0
```

Comment:
Maybe since we are using this as an example, we can add more information inside this file? just a few comments on the important areas/sections?